README.md

```diff
@@ -34,21 +34,47 @@ For more complex examples navigate to the [examples folder](./examples) where th
 Below is an example of Rust code that exposes a QObject with two properties and four invokable methods to Qt.
 
 ```rust
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize)]
+pub struct DataSerde {
```

Comment:
Is the purpose of this struct to convert the QString to a Rust std::String? Besides that, it seems redundant.

Reply:
Yes this will be required to convert from any `UniquePtr<T>` to `T` / Rust equivalent for serde. As `UniquePtr` doesn't implement (de)seralisation.

Also later on you would likely need this separate struct anyway as private members will be in the Data struct too, so eg Mutex and Channel etc might be in there.

But this is something to loop back around to after the API transition is done as we have a few ideas that might help there.

But for now, this is the route we'll have to go.

Reply:
Okay. It would help to have a brief inline comment, particularly considering this is the first example shown. Maybe something like `This struct has the fields of the Data struct below but converted to Rust types`

Reply:
Added some text.